app/src/main/java/com/duckduckgo/app/browser/WebViewRequestInterceptor.kt

```diff
@@ -106,6 +121,16 @@ class WebViewRequestInterceptor(
         return null
     }
 
+    private fun shouldAddGcpHeaders(request: WebResourceRequest): Boolean {
+        val headers = request.requestHeaders
+        return (
+            globalPrivacyControl.isGpcActive() &&
+                !headers.containsKey(GlobalPrivacyControlManager.GPC_HEADER.toLowerCase(Locale.ROOT)) &&
```

Comment:
I have a question: why are we using `toLowerCase` here? is it because when we add the headers, those are changed by something external to us? do you know why? all headers are lowercase? if it's a documented standard, why don't we change `GlobalPrivacyControlManager.GPC_HEADER` to be lowercase instead?

Reply:
Good catch, yup the headers get changed to lowercase for some reason so I did during my initial testing and never removed that. I've changed the header to always be lowercase.